pkg/network/driver/common.go

```diff
@@ -382,7 +382,7 @@ func (h *NetworkUtilsHandler) StartDHCP(nic *cache.DhcpConfig, serverAddr net.IP
 			nic.IP.Mask,
 			bridgeInterfaceName,
 			serverAddr,
-			nic.AdvertisingIPAddr,
```

Comment:
Instead of removing `StartDHCP` we can remove `serverAddr` and remove it from the `StartDHCP` params.

Reply:
If I understood it correctly, that's exactly what the next commit does.

Reply:
Right, oops:)